.drone.yml

```diff
@@ -13,6 +13,9 @@ script:
   # ensure the build works
   # this will also run gulp test
   - gulp all
+  # run tests explicitly
```

Comment:
if `gulp all` doesn't run tests anymore, remove `# this will also run gulp test` line above